tests/test_cookies.py

```diff
@@ -252,26 +252,15 @@ def test_unset(cookie, samesite='Lax'):
     assert not result_unset.cookies['baz'].same_site
 
 
-def test_cookie_expires_naive(client):
```

Comment:
If I'm not mistaken it's still allowed to set expire to a naive datetime, so this test should be kept. maybe it needs to be updated to check that expire has a timezone?

Reply:
According to the [RFC 9910](https://www.rfc-editor.org/rfc/rfc9110.html#section-5.6.7) every HTTP-date has to follow the implemented rules. That'S the reason for the deletion for the test. The test fails since, whti the new change, naive dates are no longer allowed.

Reply:
Is this still a stopping issue or is this clear now?

Reply:
sorry, I've been swamped this week and I've yet to re-look at it. Will try to find time next week

Reply:
will re-add this test making it pass.